### ISSUE TITLE
Influence of image size

### ISSUE THREAD

**Issue by robertradar** [jwyang/faster-rcnn.pytorch#794]
Hello,

is it possible to use the pretrained model on high resolution images (e.g. 4096 × 2160)? Will performance deteriorate?

**Comment by wp990329** [comment]
there's some questions.
maybe you should change the generate_archors‘s scales and ratios to make sure you have enough Positive labels
